50-tools/weblate-create-projects-oneshot.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# This script generates initial projects import script. It was used only once
+# and it could be called never more. Once regular projects maintenance will be
+# implemented, it could be deleted. This is just a source of needed code.
+
+BRANCH_ON_GITHUB=develop
+BRANCH_ON_WEBLATE=$BRANCH_ON_GITHUB
+BRANCH_SLUT=develop
```

Comment:
This variable name seems unfortunate. And I don't know what it even means. Could you change it?
(I know this is probably not written by you.)